changelog/dmd.reflocal.dd

```diff
@@ -0,0 +1,12 @@
+`ref` can now be applied to local, static, extern, and global variables
+
+For example, one can now write:
+```
+void main()
+{
+    int i;
+    ref int r = i;
+    r = 3;
+    assert(i == 3);
+}
+```
```

Comment:
I think the changelog should also mention any restrictions on local ref variables.

Reply:
The more comprehensive documentation is in the DIP, it doesn't need repeating.

Reply:
OK, I suggest a link to the DIP in the changelog.